src/RepublicanSilence/RepublicanSilenceManager.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace AppBundle\RepublicanSilence;
+
+use AppBundle\Entity\RepublicanSilence;
+use AppBundle\Repository\RepublicanSilenceRepository;
+use Psr\SimpleCache\CacheInterface;
+
+class RepublicanSilenceManager
+{
+    private const CACHE_PREFIX_KEY = 'republican_silence_';
+
+    private $repository;
+    private $cache;
+
+    public function __construct(RepublicanSilenceRepository $repository, CacheInterface $cache)
+    {
+        $this->repository = $repository;
+        $this->cache = $cache;
+    }
+
+    /**
+     * @return RepublicanSilence[]|iterable
```

Comment:
Why repeat it's iterable ? just saying RepublicanSilence[] told us it's iterable no ?